packages/odyssey-react/src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -21,6 +21,7 @@ export default {
   args: {
     label: "Checkbox label",
     value: "checkbox_value",
+    errorPrefix: "Error:",
```

Comment:
lets make a follow up JIRA to fix this bug immediately after merging this work. If you have questions lets discuss ASAP